test/e2e_test.go

```diff
@@ -254,6 +254,29 @@ func TestAttestVerify(t *testing.T) {
 
 	// Look for a specific annotation
 	mustErr(verify(pubKeyPath, imgName, true, map[string]interface{}{"foo": "bar"}, ""), t)
+
+	// Attest again with replace=true, replacing the previous attestation
+	must(attest.AttestCmd(ctx, ko, options.RegistryOptions{}, imgName, "", false, slsaAttestationPath, false,
+		"slsaprovenance", true, ftime.Duration(30*time.Second)), t)
```

Comment:
After attesting the first time with `replace=true`, let's just make sure there's 1 attestation. that should be enough for this test right now!

Reply:
Ok, I removed it. The second test did allow demonstrating the bug I mentioned though. Currently this test is incorrectly passing because replace=true is never working on `random.Image()` images.  See #1378

If you're fine with this test incorrectly passing and tracking the bug in #1378, then I think we're fine.

Reply:
Sorry to change my mind on this, but that's a good point! Maybe we leave your test in and just skip it for now, and then we can link it in the issue. Whoever picks up the issue already has the test built in :)

Reply:
Done.

Reply:
I'm not sure if there needs to be anything done to skip the test? Let me know.

Please test this PR manually on the command line considering there's only sort of a usable test.

Reply:
Actually let's copy the test into `TestAttestationReplace`, and then you can throw in a `t.Skip()` on the first line for now.